devtools/conda-envs/basic.yaml

```diff
@@ -15,7 +15,6 @@ dependencies:
   - pytest
   - pytest-cov
   - pytest-xdist
-  - codecov
```

Comment:
This should not be needed now that this runs in its own self-contained action (vs. uploading the coverage report to their servers using this command)